crypto/crypto.go

```diff
@@ -4,75 +4,77 @@ import (
 	"crypto/rand"
 	"errors"
 	"fmt"
-	"log"
-	"os"
 
 	"golang.org/x/crypto/nacl/secretbox"
 	"golang.org/x/crypto/scrypt"
+	"golang.org/x/sys/unix"
```

Comment:
@mnzt actually we should really attempt it here. We'll discuss in slack how to go about it and go over the different methods.